src/tool.cpp

```diff
@@ -171,7 +171,7 @@ void ToolCapabilities::deserializeJson(std::istream &is)
 }
 
 DigParams getDigParams(const ItemGroupList &groups,
-		const ToolCapabilities *tp)
```

Comment:
Re-adding the extra param isn't sufficient to get things working. The code that calls this function should also pass the `time_from_last_punch` again. Looking at the commit that removed it, there's much more to fix than just passing an extra argument.

Reply:
ah, you're quite right - i didn't examine the original closely enough. i'll have another look.